airflow/www/static/js/types/api-generated.ts

```diff
@@ -3002,6 +3047,66 @@ export interface operations {
       };
     };
   };
+  /**
+   * Get task dependencies blocking task from getting scheduled.
+   *
+   * *New in version 2.9.0*
```

Comment:
+1 here ... but this is generated code, correct? :-D
```suggestion
   * *New in version 2.10.0*
```

Reply:
yes this is generated from the api spec